docs/api/js.rst

```diff
@@ -305,29 +305,27 @@ event listeners, and delete the object like this:
 Example to get started with
 ===========================
 
-1. Make sure that you've installed all dependencies required by
-   :ref:`ext-http`.
+1. Follow :ref:`devenv` to get a Mopidy and Mopidy extension development
+   environment up and running.
 
-2. Create an empty directory for your web client.
+2. Follow :ref:`extensiondev` to create a new empty Mopidy extension.
 
-3. Change the :confval:`http/static_dir` config value to point to your new
-   directory.
+3. Use the :ref:`http-server-api` to make the extension host your static files,
```

Comment:
> Use the :ref:`http-server-api` to make the extension host your static files, like HTML, CSS, and JavaScript.

Use the :ref:`http-server-api` to serve a directory containing your extension's static HTML, CSS, and JavaScript files.